util/manage-page-order.p6

```diff
@@ -393,13 +467,13 @@ sub is-done(*@args) {
 }
 
 sub write-group-page($group-char, $title, $page-order) {
-    my $fname = $todir ~ "/00-Group{$group-char}.pod6";
+    my $fname = $todir ~ "/{$page-order}-Group{$group-char}.pod6";
     my $fh = open $fname, :w;
 
-    # x2588 big solid block <== using this char to make the separator
-    # x2501 solid thick horizontal line
+    # note no SUBTITLE
```

Comment:
You sure that's OK? I think Pod::To::BigPage is unhappy about stuff like this, is it actually alright in this case?

Reply:
it works on my local debian box with firefox looking at the pages